pkg/runtime/k0s/k0sctl/k0sctl_config.go

```diff
@@ -0,0 +1,135 @@
+// Copyright © 2022 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package k0sctl
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/sealerio/sealer/pkg/runtime/k0s/k0sctl/v1beta1"
+	v1 "github.com/sealerio/sealer/types/api/v1"
+	v2 "github.com/sealerio/sealer/types/api/v2"
+	utilsnet "github.com/sealerio/sealer/utils/net"
+	osi "github.com/sealerio/sealer/utils/os"
+
+	"github.com/k0sproject/dig"
+	"github.com/k0sproject/k0sctl/pkg/apis/k0sctl.k0sproject.io/v1beta1/cluster"
+	"github.com/k0sproject/rig"
+	yaml2 "gopkg.in/yaml.v2"
+)
+
+type K0sConfig struct {
+	*v1beta1.Cluster
+}
+
+// ConvertTok0sConfig convert cluster file spec host to k0sctl spec hosts.
+func (c *K0sConfig) ConvertTok0sConfig(clusterFile *v2.Cluster) error {
+	return c.convertIPVSToAddress(clusterFile)
+}
+
+func (c *K0sConfig) convertIPVSToAddress(clusterFile *v2.Cluster) error {
```

Comment:
I think the logic here seems to be parsing the host configuration information in the *v2.Cluster structure. Do we need to rename this function? `convertClusterHosts` ?